tests/unit/components/QueryBuilder.spec.ts

```diff
@@ -40,43 +47,45 @@ describe( 'QueryBuilder.vue', () => {
 
 	it( 'passes the selected property back to the PropertyLookup', async () => {
 		const property = { label: 'postal code', id: 'P123' };
-		const wrapper = shallowMount( QueryBuilder, {
-			store: newStore( {
-				property: jest.fn().mockReturnValue( property ),
-			} ),
-			localVue,
-		} );
+		const conditionIndex = 0;
+		const wrapper = shallowMount( QueryBuilder, { store: newStore(), localVue } );
```

Comment:
We probably want to explicitly inject the property getter here (can be very simple like `() => () => ({ label: 'postal code', id: 'P123' })`), so that if the dummy code in the default store is changed, this test won't suddenly start to fail. As it is currently, this test has a hidden dependency on the mock data in the `newStore()` method.

Reply:
Done